spec/support/test_helpers.rb

```diff
@@ -30,7 +30,7 @@ def agfs_claim_type_objects
       end
 
       def lgfs_claim_type_objects
-        [Claim::LitigatorClaim, Claim::InterimClaim, Claim::TransferClaim]
+        [Claim::LitigatorClaim, Claim::InterimClaim, Claim::TransferClaim, Claim::LitigatorHardshipClaim]
```

Comment:
going to need rebasing as I moved this to `spec/support/shared_examples_for_claim_types.rb`